programs/bpf_test_utils/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "solana-bpf-test-utils"
```

Comment:
This could be part of the sdk to avoid releasing another crate

Reply:
Tried it, but it results in a dependency cycle between solana-runtime and solana-sdk

Reply:
I think this can just go directly into runtime.  We'll need these mocks from program-based tests as well.   It can't go into sdk because InvokeContext/etc are runtime-only things (which I think is a good spot for them).

I'd even like to see `bpf_test_utils` landed sooner, in a separate PR.  We'll want the mocks in 1.4 as well, so a separate PR will be easier to backporting

Reply:
`bpf_test_utils` is now a module in `solana_runtime`, should I copy that to an extra pull request and revert the API changes there?

Reply:
Be cleaner if you broke it out

Reply:
Done: https://github.com/solana-labs/solana/pull/13203